.github/ISSUE_TEMPLATE/--new-addon.md

```diff
@@ -1,16 +1,37 @@
 ---
 name: "➕ New addon"
-about: Suggest a new addon
+about: 'Suggest a new addon'
 title: ''
 labels: 'type: enhancement, new addon, scope: addon'
 assignees: ''
-
 ---
 
-**Describe the new addon**
+<!--
+
+READ BEFORE CREATING!
+
+- Avoid creating duplicates! Read the FAQ page and search through Issues and Discussions before creating one.
+- Try to do a simple but descriptive title, and include detailed information in here.
+- Make sure to use the latest version of Scratch Addons.
+- Asking for help should be done on Discussions or our Discord server. Return to the last page for the links.
+
+FAQ: https://scratchaddons.com/faq
+Return to the last page: https://github.com/ScratchAddons/ScratchAddons/issues/new/choose
+
+-->
+
+### Is your feature request related to a problem? Please describe.
```

Comment:
In this file replace 'feature request' with 'addon request'

Reply:
Huh, good idea.